src/settingsView.ts

```diff
@@ -202,14 +264,20 @@ function htmlForWebview(vscode: vscodeTypes.VSCode, extensionUri: vscodeTypes.Ur
     <html lang="en">
     <head>
       <meta charset="UTF-8">
-      <meta http-equiv="Content-Security-Policy" content="default-src 'none'; style-src ${webview.cspSource}; script-src 'nonce-${nonce}';">
+      <meta http-equiv="Content-Security-Policy" content="default-src 'none'; style-src 'unsafe-inline' ${webview.cspSource}; script-src 'nonce-${nonce}';">
```

Comment:
Do you really need `unsafe-inline`?

Reply:
Well, we can use a png icon, or find a different svg. [Here's an example](https://github.com/grommet/grommet-icons/blob/master/public/img/chrome.svg?short_path=692d202).